plugins/imjournal/imjournal.c

```diff
@@ -520,24 +520,23 @@ persistJournalState(void)
 	DEFiRet;
 	FILE *sf = NULL; /* state file */
 	char tmp_sf[MAXFNAME];
-	size_t n;
 
 	DBGPRINTF("Persisting journal position, cursor: %s, at head? %d\n",
 			  journalContext.cursor, journalContext.atHead);
 
 	/* first check that we have valid cursor */
 	if (!journalContext.cursor) {
+		DBGPRINTF("Journal cursor is not valid, ok...\n");
```

Comment:
More precisely we do not have any cursor at all. I, as journald API user, cannot in fact decide by myself whether cursor is valid (journald API call is needed for that)

Reply:
:)